platform/drm/cog-platform-drm.c

```diff
@@ -352,6 +352,22 @@ check_drm(void)
     return supported;
 }
 
+static int32_t
+find_crtc_for_encoder(const drmModeRes *resources, const drmModeEncoder *encoder) {
```

Comment:
You need to move the opening brace to its own line to make the style checker happy:

```suggestion
find_crtc_for_encoder(const drmModeRes *resources, const drmModeEncoder *encoder)
{
```